libs/liquidity/src/lib/liquidity-manager.tsx

```diff
@@ -0,0 +1,132 @@
+import { useRef, useMemo } from 'react';
+import {
+  AsyncRenderer,
+  ResizableGrid,
+  ResizableGridPanel,
+  Tab,
+  Tabs,
+} from '@vegaprotocol/ui-toolkit';
+
+import type { AgGridReact } from 'ag-grid-react';
+import { LiquidityTable } from './liquidity-table';
+import type { MarketLiquidity } from './__generated__/MarketLiquidity';
+import { useQuery } from '@apollo/client';
+import classNames from 'classnames';
+import { t } from '@vegaprotocol/react-helpers';
+import { LiquidityProvisionStatus } from '@vegaprotocol/types';
+import {
+  MARKET_LIQUIDITY_QUERY,
+  useLiquidityProvision,
+} from './liquidity-data-provider';
+
+export const SISKA_NETWORK_PARAMETER = 'market.liquidity.stakeToCcySiskas';
+
+interface LiquidityManagerProps {
+  partyId?: string;
+  marketId: string;
+}
+
+export const LiquidityManager = ({
+  partyId,
+  marketId,
+}: LiquidityManagerProps) => {
+  const gridRef = useRef<AgGridReact | null>(null);
+  const {
+    data: marketLiquidityData,
+    loading,
+    error,
+  } = useQuery<MarketLiquidity>(MARKET_LIQUIDITY_QUERY, {
+    variables: useMemo(() => ({ marketId }), [marketId]),
+  });
+
+  const { liquidityProviders, suppliedStake, targetStake, code, symbol } =
+    useLiquidityProvision({ data: marketLiquidityData });
+
+  const wrapperClasses = classNames(
+    'h-full max-h-full',
+    'flex flex-col',
+    'text-ui-small',
+    'gap-8'
+  );
+  const titleClasses =
+    'sm:text-lg md:text-xl lg:text-2xl flex items-center gap-4 whitespace-nowrap';
+
+  const myLpEdges = useMemo(
+    () => (liquidityProviders || []).filter((e) => e.party === partyId),
+    [liquidityProviders, partyId]
+  );
+  const activeEdges = useMemo(
+    () =>
+      (liquidityProviders || []).filter(
+        (e) => e.status === LiquidityProvisionStatus.STATUS_ACTIVE
+      ),
+    [liquidityProviders]
+  );
+  const inactiveEdges = useMemo(
+    () =>
+      (liquidityProviders || []).filter(
+        (e) => e.status !== LiquidityProvisionStatus.STATUS_ACTIVE
+      ),
+    [liquidityProviders]
+  );
+
+  const getActiveDefaultId = () => {
+    if (activeEdges?.length) return 'active';
+    else if (inactiveEdges?.length > 0) return 'inactive';
+    return 'active';
+  };
+  const itemClass =
+    'min-w-min w-[120px] whitespace-nowrap pb-3 px-4 border-l border-neutral-300 dark:border-neutral-700';
+  const itemHeading = 'text-neutral-400';
+  return (
+    <AsyncRenderer loading={loading} error={error} data={liquidityProviders}>
+      <header className="w-screen xl:px-4 pt-4 border-b border-neutral-300 dark:border-neutral-700">
```

Comment:
chore (blocking): This UI should be the concern of the app and not the lib as it deals with page layout. Ideally the lib should only export things that are app agnostic. In this case only the table component and the data provider/hook should be exported from the lib. Things like market id and party id should be passed in as props from the app.

Reply:
updated